features/fork.feature

```diff
@@ -154,7 +154,7 @@ Scenario: Related fork already exists
     Then the exit status should be 1
     And the stderr should contain exactly:
       """
-      Error creating fork: No git remote with name origin\n
+      Error creating fork: No git remote with name origin or upstream\n
```

Comment:
You can maybe change the error message to: "no origin git remote found". That doesn't imply that the remote needs to be literally named "origin".